.circleci/config.yml

```diff
@@ -53,6 +53,7 @@ commands:
             echo 'export CARGO_INCREMENTAL=0' >> $BASH_ENV
             echo 'export CI_TIMEOUT="timeout 40m"' >> $BASH_ENV
             echo 'export CC=gcc' >> $BASH_ENV
+            echo 'export FIL_PROOFS_PARAMETER_CACHE=$HOME/.proof-params' >> $BASH_ENV
```

Comment:
Whats wrong with the default path?

Reply:
it's in the tmp directory, so this is more easily accessed and persisted (it's needed when running certain commands, but not immediately needed now in the CI). Just a good idea to set imo